libs/shareProcessor.js

```diff
@@ -85,6 +85,7 @@ module.exports = function(logger, poolConfig){
             redisCommands.push(['rename', coin + ':shares:roundCurrent', coin + ':shares:round' + shareData.height]);
             redisCommands.push(['rename', coin + ':shares:timesCurrent', coin + ':shares:times' + shareData.height]);
             redisCommands.push(['sadd', coin + ':blocksPending', [shareData.blockHash, shareData.txHash, shareData.height, shareData.worker, dateNow].join(':')]);
+            redisCommands.push(['hset', 'Allblocks', coin + "-" + shareData.height, [shareData.blockHash, shareData.txHash, shareData.height, shareData.worker, dateNow].join(':')]); //used for block stat
```

Comment:
This isn't needed for last block mined, it was something I added to get all blocks from a single api call

The last block is just sort/slice of it.stats.pools[pool.]pending.blocks and confirmed.blocks
https://github.com/madbuda/z-nomp-mb/blob/25c761865b1a30cd1a7f286a71fc32cd5f2c364a/libs/stats.js#L391